core/src/main/java/com/epam/esm/dao/entity/Tag.java

```diff
@@ -0,0 +1,67 @@
+package com.epam.esm.dao.entity;
+
+import java.io.Serializable;
+import java.util.HashSet;
+import java.util.Set;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+/** Entity class for creating {@code Tag} objects. */
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "tag")
+public class Tag implements Serializable {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@EqualsAndHashCode.Exclude
+	private long id;
+
+	@Column(name = "name")
+	private String name;
+
+	public Tag(String name) {
+		this.name = name;
+	}
+
+	public Tag(long id, String name) {
```

Comment:
could this constructor be replaced by Lombok annotation?

Reply:
The annotation _@AllArgsConstructor_ will also include _private Set<Certificate> certificates_

and  _@RequredArgsConstructor_ requires to make _long id_, _String name_ as final. Is it OK to have them like that?